### DISCUSSION TITLE
Bring "fullscreen" view also to Projects (beta)

### DISCUSSION THREAD

**Discussion by hennrunnel** [community/community#5749] [Projects and Issues]
Please bring the "fullscreen" view available in old Projects (see screenshot below) to the new one. The fullscreen view is a helpful way to increase the screen real estate, esp for those working on a laptop. 

![image](https://user-images.githubusercontent.com/11610902/134176177-24dadcab-9908-4438-8d4b-d0ea4eb8bf6c.png)

**Comment by RalfJung** [comment]
Has the new top bar now been landed without opt-out and without fullscreen project support? Congrats Github, you made my screen 30% less useful. :(

I guess now we know how much GH cares for the feedback that they are asking for in their betas.

**Comment by jantmanAtCox** [comment]
This is... over 2 and a half _years_ old now? Any news?

**Comment by ePaul** [comment]
On the "bright side", with the latest GH Enterprise the full screen mode also disappeared from the classic projects, so there is no feature disparity anymore. :frowning_face:

**Comment by lukehefson** [comment]
👋 Projects friends! Every Monday I share a project board with my team in a zoom call. And every Monday I miss that fullscreen button 😿.
